README.md

```diff
@@ -1,121 +1,134 @@
+> [!NOTE]
+> Versions 1.2.x of the SDK support Internet Explorer >=9 and other older browsers, as well as Node.js >=8.17.
 
-If you require support for older browsers and Node.js, you can use the security-maintained version 1 of the library. Install version 1 via [CDN link](https://cdn.ably.com/lib/ably.min-1.js), or from npm with `npm install ably@1 --save`. It supports IE versions 9 or newer, older versions of major browsers, and Node.js 8.17 or newer. Note that version 1 will only receive security updates and critical bug fixes, and won't include any new features.
+> [!IMPORTANT]
+> SDK versions < 1.2.36 will be [deprecated](https://ably.com/docs/platform/deprecate/protocol-v1) from November 1, 2025.
 
-For complete API documentation, see the [Ably documentation](https://www.ably.com/docs).
+---
 
 ## Installation
 
-### Node.js
+The NodeJS SDK is available as an [NPM module](https://www.npmjs.com/package/ably). To get started with your project, install the package:
 
-    npm install ably --save
+```sh
+npm install ably
+```
 
-and require as:
+Run the following to instantiate a client:
 
 ```javascript
-var Ably = require('ably');
+const Ably = require('ably');
+const realtime = new Ably.Realtime({ key: apiKey });
 ```
 
-For usage, jump to [Using the Realtime API](#using-the-realtime-api) or [Using the REST API](#using-the-rest-api).
+### Webpack installation
 
-#### Serverside usage with Webpack
+The Ably Pub/Sub SDK includes support for Webpack compiling browsers.
 
-If you are using a version older than 1.2.5 you will need to add 'ably' to `externals` in your Webpack config to exclude it from Webpack processing, and require and use it in as a external module using require('ably') as above.
+<details>
+<summary>Webpack installation details.</summary>
 
-### For browsers
+If you're compiling for the browser, Webpack will resolve `ably` from `node_modules` automatically when included in your `package.json`. You can then:
 
-Include the Ably library in your HTML:
-
-```html
-<script src="https://cdn.ably.com/lib/ably.min-2.js"></script>
+```javascript
+require('ably');
+// or, for ES6/TypeScript:
+import * as Ably from 'ably';
 ```
 
-The Ably client library follows [Semantic Versioning](http://semver.org/). To lock into a major or minor version of the client library, you can specify a specific version number such as https://cdn.ably.com/lib/ably.min-2.js for all v2._ versions, or https://cdn.ably.com/lib/ably.min-2.6.js for all v2.6._ versions, or you can lock into a single release with https://cdn.ably.com/lib/ably.min-2.6.3.js. Note you can load the non-minified version by omitting `.min` from the URL such as https://cdn.ably.com/lib/ably-2.js. See https://github.com/ably/ably-js/tags for a list of tagged releases.
+With `target: 'browser'`, Webpack uses the browser-compatible CommonJS build by default.
+
+If needed (for example with custom targets):
+
+- Webpack 5: To resolve Ably correctly, add this alias to your webpack configuration:
```

Comment:
not needed in this PR but someone should revisit this webpack section, i don't believe you would actually need to do any of this since ably-js v2